### PR TITLE
Organize & Initialize a new Ember project

### PR DESCRIPTION
**Pull request by BT-Creator**

### CODE REVIEW

File: misc/ember map/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
```
[the remark this answers is not in the document]
Comment (BT-Creator) [reply misc/ember map/.editorconfig:13]: That was the init's work ¯\_(ツ)_/¯

[the remark this answers is not in the document]
Comment (BT-Creator) [reply misc/ember map/.editorconfig:13]: We can change it later if needed

[the remark this answers is not in the document]
Comment (samvdkris) [reply misc/ember map/.editorconfig:13]: Hmmmmm alright then

[the remark this answers is not in the document]
Comment (samvdkris) [reply misc/ember map/.editorconfig:13]: But damn, I think the only places where I would use double spaces is YAML, and *maybe* JSON. JavaScript? No, cursed. Bad. Evil.

**Pull request merged**